src/Elcodi/Admin/PluginBundle/AdminPluginBundle.php

```diff
@@ -30,6 +30,6 @@ class AdminPluginBundle extends Bundle
      */
     public function getContainerExtension()
     {
-        return null;
+        return;
```

Comment:
I think it's more explicit to have a `return null`, but maybe that's just my own opinion.